.travis.yml

```diff
@@ -1,19 +1,16 @@
-# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 os:
-  - linux
-  - osx
+    - linux
+    - osx
 julia:
-  - release
-  - nightly
+    - nightly
```

Comment:
Better make this
```
- 0.5
- nightly
```
because those two are already different from one another (and the differences will keep growing).